### PR TITLE
Pull request for WAZO-2936-room-user-uuid-query

### PR DESCRIPTION
**Pull request by fblackburn1**
Depends-on: https://github.com/wazo-platform/wazo-chatd-client/pull/10

### CODE REVIEW

File: wazo_chatd/plugins/api/api.yml [hunk 1]
```diff
@@ -134,3 +134,11 @@ parameters:
...
     type: integer
     description: Number of items to skip over in the list. Useful for pagination.
+  user_uuid_query:
+    required: false
+    name: user_uuid
+    in: query
+    type: array
+    items:
+      type: string
+    description: Filter by user_uuid. Many uuid can be specified. Each uuid MUST be separated by a comma (,).
```
Comment (bloom1) [inline wazo_chatd/plugins/api/api.yml:144]: Just to be clearer - wouldn't it be nice to specify which operation is made with the user_uuids? A logical OR or a logical AND?

Comment (fblackburn1) [reply wazo_chatd/plugins/api/api.yml:144]: :+1: Yes !!

**Review by bloom1 — APPROVED** [review]

**Comment by bloom1** [comment]
recheck

**Comment by fblackburn1** [comment]
regate

**Pull request merged**